src/memorypad.cpp

```diff
@@ -253,20 +253,20 @@ struct MemoryPadTrackpad : OpaqueWidget {
     }
   }
 
-  void draw(const DrawArgs& args) override {
+  void drawLayer(const DrawArgs& args, int layer) override {
+    if (layer != 1 || !module) return;
+
     OpaqueWidget::draw(args);
```

Comment:
This should be removed right? (Also, I wasn't sure if it should go at the start or end. Is it not an issue to have the call to super never happen in the case then the if you added in ln 257 returns early?)

Reply:
Yeah, my mistake. I'll delete that line